doc/api/os.md

```diff
@@ -26,6 +26,17 @@ The operating system-specific end-of-line marker.
 * `\n` on POSIX
 * `\r\n` on Windows
 
+## `os.availableParallelism()`
```

Comment:
should this be marked as experimental?

Reply:
I don't think so in this case.

Reply:
Let me expand on that a bit. The API signature is very simple and seems unlikely to change. If it does somehow change, it would almost certainly be a semver-minor addition.

Reply:
that's what I thaught, but I was mainly asking about the existence/inexistence of an internal rule or process